tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -309,7 +325,9 @@ void TRTEngineOp::ExecuteCalibration(OpKernelContext* ctx,
     const auto device_tensor =
         calib_res->device_tensors_.at(i).AccessTensor(ctx);
     CHECK_EQ(t.TotalBytes(), device_tensor->TotalBytes());
-    input_data.emplace(StrCat(kInputPHName, i), data_address);
+    input_data.emplace(
+        StrCat(prefixes.kInputPHName, static_engine_ ? i : input_node_ids_[i]),
```

Comment:
I don't understand, why not just use the same id `i`? ConvertGraphDefToEngine() uses the same `i` from the Placeholder/Identity node's name suffix right?

Same question below.